src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
```

Comment:
The use of React Router for saving page and perPage parameters in the URL is missing. Please implement this feature as per the task requirements.